README.md

```diff
@@ -50,7 +50,7 @@ If you've downloaded the repo and are editing Mattermost documentation on your l
 2. Install [pipenv](https://docs.pipenv.org/): `pip install pipenv`. 
 3. `cd` into the cloned repository.
 4. Install the required packages: `pipenv install`.
-5. Build the doc set using `make html`. This generates files in `/build` directory.
+5. Build the doc set using `make HTML`. This generates files in `/build` directory.
```

Comment:
Would you be open to confirming whether you experienced a build problem using "html"? The [Sphinx documentation](https://www.sphinx-doc.org/en/master/usage/quickstart.html) specifies `make html` which is why we've included it in that casing.